### PR TITLE
TestKit backend: except txMeta as Cypher types

### PR DESCRIPTION
**Pull request by robsdedude**
Depends on:
 * https://github.com/neo4j-drivers/testkit/pull/538

### CODE REVIEW

File: Neo4j.Driver/Neo4j.Driver.Tests.TestBackend/Protocol/Session/SessionRun.cs [hunk 1]
```diff
@@ -23,46 +23,11 @@ public class SessionRunType : BaseSessionType
...
-            if (data.txMeta.Count > 0) 
-                configBuilder.WithMetadata(data.txMeta);
-        }
-
         public override async Task Process()
         {
             var newSession = (NewSession)ObjManager.GetObject(data.sessionId);
-            IResultCursor cursor = await newSession.Session.RunAsync(data.cypher, ConvertParameters(data.parameters), TransactionConfig).ConfigureAwait(false);
+            IResultCursor cursor = await newSession.Session
+                .RunAsync(data.cypher, CypherToNativeObject.ConvertDitctionaryToNative(data.parameters), data.TransactionConfig).ConfigureAwait(false);
```
Comment (RichardIrons-neo4j) [inline Neo4j.Driver/Neo4j.Driver.Tests.TestBackend/Protocol/Session/SessionRun.cs]:
```
ConvertDitctionaryToNative
         ^
```
Typo here, an extra `t` in the name 🙂

**Review by RichardIrons-neo4j — COMMENTED** [review]
Nothing except a typo, but also I think run a formatter on it to apply the .editorconfig settings.

**Review by RichardIrons-neo4j — APPROVED** [review]
All good.

**Comment by robsdedude** [comment]
The same 5 tests as on `5.0` are failing.

**Pull request merged**